pkg/logger/filelogger.go

```diff
@@ -0,0 +1,48 @@
+package logger
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"runtime"
+	"time"
+)
+
+// FileLogSave is a logging system that
+// saves log data in a new log file
+type FileLogSave struct {
+	LogFileName string
+}
+
+// SaveLogData in FileLogSave will save any logged
+// info/erros into a log file on the local file system
+func (logSave FileLogSave) SaveLogData(data string) {
+	logfile := logSave.getLogFile(logSave.LogFileName)
+	defer logfile.Close()
+	log.SetOutput(logfile)
+	log.Print(data)
+}
+
+func (logSave FileLogSave) getLogFile(file string) *os.File {
+	directory := logSave.defaultDirectory()
+	filename := logSave.defaultFile()
+	fullPath := directory + filename
```

Comment:
Could be an idea to cache the path? Just lighten the I/O load ever so slightly, may not matter unless we start logging loads...